library/src/lib/calendar/calendar.component.ts

```diff
@@ -514,32 +576,35 @@ export class CalendarComponent implements OnInit, OnDestroy, AfterViewChecked, C
         return retVal;
     }
 
-    // Functions that handle calendar navigation
+    /** @hidden */
     goToPreviousMonth() {
         this.setCurrentMonth(this.date.getMonth() - 1);
         this.selectedMonth = this.month;
         this.constructCalendar();
     }
 
+    /** @hidden */
     goToNextMonth() {
         this.setCurrentMonth(this.date.getMonth() + 1);
         this.selectedMonth = this.month;
         this.constructCalendar();
     }
 
+    /** @hidden */
     loadNextYearsList() {
         this.calendarYearsList = [];
         this.firstYearCalendarList += 12;
         this.constructCalendarYearsList();
     }
 
+    /** @hidden */
     loadPrevYearsList() {
         this.calendarYearsList = [];
         this.firstYearCalendarList -= 12;
         this.constructCalendarYearsList();
     }
 
-    // Functions that handle selection (day, month, year)
+    /** @hidden */
```

Comment:
I think this should be exposed, people could use it to programatically select a date, no?